presto-main/src/main/java/io/prestosql/operator/TableWriterOperator.java

```diff
@@ -89,7 +90,8 @@ public TableWriterOperatorFactory(
             this.planNodeId = requireNonNull(planNodeId, "planNodeId is null");
             this.columnChannels = requireNonNull(columnChannels, "columnChannels is null");
             this.pageSinkManager = requireNonNull(pageSinkManager, "pageSinkManager is null");
-            checkArgument(writerTarget instanceof CreateTarget || writerTarget instanceof InsertTarget, "writerTarget must be CreateTarget or InsertTarget");
+            checkArgument(writerTarget instanceof CreateTarget || writerTarget instanceof InsertTarget || writerTarget instanceof TableWriterNode.RefreshMVTarget,
+                    "writerTarget must be CreateTarget or InsertTarget");
```

Comment:
Enhance the message to say:
'writerTarget must be CreateTarget, InsertTarget or RefreshMVTarget'